tests/zfs-tests/tests/functional/cli_root/zpool_split/zpool_split_resilver.ksh

```diff
@@ -0,0 +1,97 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2018, Nexenta Systems, Inc.  All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+# 'zpool split' should fail if resilver in progress for a disk
+#
+# STRATEGY:
+# The first scenario:
+# 1. Create a mirror pool
+# 2. Offline the first VDEV
+# 3. Put some data
+# 4. Online the first VDEV
+# 5. Verify 'zpool split' must fail
+#
+# The second scenario:
+# 1. Create a mirror pool
+# 2. Offline the second VDEV
+# 3. Put some data
+# 4. Online the second VDEV
+# 5. Verify 'zpool split' must fail
+#
+
+verify_runnable "both"
+
+function cleanup
+{
+	destroy_pool $TESTPOOL
+	destroy_pool $TESTPOOL2
+	rm -f $DEVICE1 $DEVICE2
+}
+
+function setup_mirror
+{
+	truncate -s $DEVSIZE $DEVICE1
+	truncate -s $DEVSIZE $DEVICE2
+	log_must zpool create -f $TESTPOOL mirror $DEVICE1 $DEVICE2
+}
+
+function zpool_split #disk
+{
+	typeset disk=$1
+
+	setup_mirror
+
+	log_must zpool offline $TESTPOOL $disk
+
+	# Create 2G of additional data
+	mntpnt=$(get_prop mountpoint $TESTPOOL)
+	log_must file_write -b 2097152 -c 1024 -o create -d 0 -f $mntpnt/biggerfile
+	log_must sync
+
+	log_must set_tunable64 zfs_scan_vdev_limit $ZFS_SCAN_VDEV_LIMIT_SLOW
+
+	log_must zpool online $TESTPOOL $disk
+
+	log_must is_pool_scrubbing $TESTPOOL true
```

Comment:
`s/is_pool_scrubbing/is_pool_resilvering/`, failure log:

```
Test: /usr/share/zfs/zfs-tests/tests/functional/cli_root/zpool_split/zpool_split_resilver (run as root) [00:36] [FAIL]
23:49:07.23 ASSERTION: Verify 'zpool split' will fail if resilver in progress for a disk
23:49:07.35 SUCCESS: zpool create -f testpool mirror /mnt/device-1 /mnt/device-2
23:49:07.40 SUCCESS: zpool offline testpool /mnt/device-1
23:49:24.86 SUCCESS: file_write -b 2097152 -c 1024 -o create -d 0 -f /testpool/biggerfile
23:49:24.95 SUCCESS: sync
23:49:24.95 SUCCESS: set_tunable64 zfs_scan_vdev_limit 131072
23:49:29.64 SUCCESS: zpool online testpool /mnt/device-1
23:49:29.65 NOTE: scan: resilver in progress since Tue Sep 11 23:49:25 2018
23:49:29.65 ERROR: is_pool_scrubbing testpool true exited 1
```

I would also suggest using `zinject -d <device> -D 50:1 $TESTPOOL` on both devices before `zpool online` and instead of asserting `is_pool_resilvering` do something like `while ! is_pool_resilvering { sleep 1 }` so we don't accidentally fail if the resilver is slow to kick in.